packages/hdx_zed/extension.toml

```
@@ -0,0 +1,11 @@
+id = "hdx"
+name = "hdx"
+version = "0.0.1"
+schema_version = 1
+authors = ["Keith Cirkel (https://keithcirke.co.uk)"]
```

You should add your name here! But also
```suggestion
authors = ["Keith Cirkel (https://keithcirkel.co.uk)"]
```

---

I actually copied this from the vscode package json. So possibly typo there too, will fix up. 

And yeah dude, thank you will do! 🥰